bin/md2gslides.js

```diff
@@ -124,9 +125,23 @@ function authorizeUser() {
     // application/utility such as this.  Of course, in such cases the "secret" is
     // actually publicly known; security depends entirely on the secrecy of refresh
     // tokens, which effectively become bearer tokens.
+
```

Comment:
Something that might help maintainers to get this merged is focusing on details, let's have this code more resilient. I think we have couple of things we could improve: :hugs: